scripts/table_diff/changes/changes.go

```diff
@@ -20,9 +20,24 @@ type change struct {
 	Breaking bool   `json:"breaking"`
 }
 
+type columnType int
+
+const (
+	columnTypePK columnType = 1 << iota
```

Comment:
Based on https://github.com/cloudquery/plugin-sdk/blob/37254f3ae99eb02bec824a211dd9e3b7869b55dd/plugins/source/templates/table.md.go.tpl#L41 columns can be both PK and Incremental (though I don't think we have those yet), so using a bit mask